articles/resources/relnotes.md

```diff
@@ -16,6 +16,21 @@ For installation instructions, please refer to the [install guide](xref:microsof
 
 For update instructions, please refer to the [update guide](xref:microsoft.quantum.update).
 
+## Version 0.xx.xxxx.xxx
+
+*Release date: July 1st, 2020*
+
+This release contains the following:
+
+- New qdk-chem chemistry format conversion tool 
+- New synthesis functionality in QuantumLibraries
+- IQ# now allows arguments to %simulate and %estimate
+- New phase display options in IQ#
+- Updated conda package support for IQ# installation to support local installation to a conda environment
```

Comment:
@rmshaffer, @KittyYeungQ: Should we have a brief documentation page on this feature, perhaps marked as an advanced page or something?

```suggestion
- IQ# is now provided as a [`conda` package](https://anaconda.org/quantum-engineering/iqsharp) for IQ# installation to support local installation to a conda environment.
```

Reply:
I've actually been working on updating the IQ#/Python installation guide to make conda the recommended path, and mark the existing one as "advanced". Here's preview renderings of what I have so far:

- https://review.docs.microsoft.com/en-us/quantum/quickstarts/install-jupyter?branch=pr-en-us-923&tabs=tabid-conda
- https://review.docs.microsoft.com/en-us/quantum/quickstarts/install-python?branch=pr-en-us-923&tabs=tabid-conda
- https://review.docs.microsoft.com/en-us/quantum/quickstarts/update?branch=pr-en-us-923&tabs=tabid-conda

Draft PR here: https://github.com/MicrosoftDocs/quantum-docs-pr/pull/923